scripts/generate-regression-tests.sh

```diff
@@ -90,12 +91,13 @@ replace-tests() {
 
     if [ -d $testdir ]
     then
-        rm $testdir/test-*
+        rm $testdir/!(*.golden|Makefile)
     else
         mkdir $testdir
         echo "include \$(CURDIR)/../include.mk" > $testdir/Makefile
         echo "" >> $testdir/Makefile
         echo "test-%.sh.out: \$(TEST_DIR)/test-%-*" >> $testdir/Makefile
+        # TODO: do we want to "make golden" here?
```

Comment:
I think it's OK to do nothing here. The tests will fail when the pull request is added.